highfidelity/hifi
# Trying to fix dependency shutdown

https://highfidelity.manuscript.com/f/cases/16254/DependencyManager-should-validate-that-a-dependency-is-actually-destroyed-when-it-s-asked

Test plan:
- Verify that the tablet and audio still work.  Verify that you can run and stop scripts.
- Verify that you don't crash on shutdown.  After shutdown, check your logs.  Near the end, you should see "DependencyManager::destroy(): class ScriptEngines was not properly destroyed!" but you shouldn't see any other warnings about `DependencyManager::destroy()`.
